### PR TITLE
resource example

### PR DESCRIPTION
**Pull request by fosterlynn**

### CODE REVIEW

File: examples/resource-examples.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Example: Resources
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - farm: https://farm.example/
+    ose: https://www.opensourceecology.org
```
Comment (elf-pavlik) [inline examples/resource-examples.yaml]: misses trailing `/`

File: examples/resource-examples.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    name: Tractor - LifeTrac 6
+    trackingIdentifier: 889jcd00s # the serial number assigned by the manufacturer
+    resourceConformsTo: ose:822b6baf-d66f-45a0-98f2-57450e2aec23
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  - '@id': farm:07819789-dd51-44c3-b35c-9210165bc832
+    '@type': EconomicResource
+    name: 12 kg case organic carrots
```
Comment (elf-pavlik) [inline examples/resource-examples.yaml]: can we start without cases, just 600kg of carrots and later we can add variant which expresses 12kg cases in machine readable way

File: examples/resource-examples.yaml (hunk 1 above)
Comment (fosterlynn) [reply examples/resource-examples.yaml]: done thanks

File: examples/resource-examples.yaml (hunk 2 above)
Comment (fosterlynn) [reply examples/resource-examples.yaml]: done

**Comment by fosterlynn** [comment]
Fixed/changed per review.  

Also made a list of classifiedAs links, could you please check and see if the format/syntax is right?  Thought it might be good to get that into an example.  Looked for agrivoc classifications, which would be better, but couldn't parse their website in a reasonable amount of time.

File: examples/resource-examples.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  - '@id': farm:07819789-dd51-44c3-b35c-9210165bc832
+    '@type': EconomicResource
+    name: Loose organic carrots
+    trackingIdentifier: 54988 # the lot number assigned by the farm
+    classifiedAs: 
+      - https://www.wikidata.org/wiki/Q81 # carrot
+      - https://www.wikidata.org/wiki/Q9566427 # organic certification
```
Comment (elf-pavlik) [inline examples/resource-examples.yaml]: Maybe let's not experiment with it at this moment? Some agent may offer organic certification for produce and that service could have this classification. Just using `Q81` seems enough to get this use case started.

**Comment by fosterlynn** [comment]
removed organic cert

**Pull request merged**